### PR TITLE
[FIZZ-153] Sync user-activities

### PR DESCRIPTION
**Pull request by lalrg**

### CODE REVIEW

File: src/google-classroom-extractor/google_classroom_extractor/mapping/user_submission_activities.py [hunk 1]
```diff
@@ -117,6 +121,8 @@ def submissions_to_user_submission_activities_dfs(
                 "gradeChangeType",
                 "courseId",
                 "actorUserId",
+                "CreateDate",
+                "LastModifiedDate"
```
Comment (bradbanister) [inline src/google-classroom-extractor/google_classroom_extractor/mapping/user_submission_activities.py:125]: You'll want to add empty "SourceCreateDate" and "SourceLastModifiedDate" fields as well.  See for example users.py lines 80-81

File: src/google-classroom-extractor/tests/mapping/user_submission_activities/test_user_submission_activities.py [hunk 2]
```diff
@@ -72,7 +76,7 @@ def it_should_have_correct_shape(assignment_submissions_dicts):
         row_count, column_count = submissions_df.shape
 
         assert row_count == 1
-        assert column_count == 11
+        assert column_count == 13
```
Comment (bradbanister) [inline src/google-classroom-extractor/tests/mapping/user_submission_activities/test_user_submission_activities.py:79]: Also need to assert new state_row values specifically.

File: src/google-classroom-extractor/tests/mapping/user_submission_activities/test_user_submission_activities.py [hunk 3]
```diff
@@ -129,7 +135,7 @@ def it_should_have_correct_shape(assignment_submissions_dicts):
         row_count, column_count = submissions_df.shape
 
         assert row_count == 2
-        assert column_count == 11
+        assert column_count == 13
```
Comment (bradbanister) [inline src/google-classroom-extractor/tests/mapping/user_submission_activities/test_user_submission_activities.py:138]: Also need to assert new state_row and grade_row values specifically.

**Review by bradbanister — CHANGES_REQUESTED** [review]
Minor changes needed for FIZZ-153: Sync user activities. 

Also, please review my commits touching up FIZZ-154: Assignments and FIZZ-155: Submissions.

File: src/google-classroom-extractor/google_classroom_extractor/mapping/user_submission_activities.py (hunk 1 above)
Comment (lalrg) [reply src/google-classroom-extractor/google_classroom_extractor/mapping/user_submission_activities.py:125]: ✔️

File: src/google-classroom-extractor/tests/mapping/user_submission_activities/test_user_submission_activities.py (hunk 3 above)
Comment (lalrg) [reply src/google-classroom-extractor/tests/mapping/user_submission_activities/test_user_submission_activities.py:138]: ✔️

File: src/google-classroom-extractor/tests/mapping/user_submission_activities/test_user_submission_activities.py [hunk 4]
```diff
@@ -35,6 +35,8 @@
 GRADE_CHANGE_TYPE = "16"
 GRADE_TIMESTAMP = "17"
 ACTOR_USER_ID = "18"
+LAST_MODIFIED_DATE = "01-01-2020"
+CREATE_DATE = "01-01-2020"
```
[the remark this answers is not in the document]
Comment (lalrg) [reply src/google-classroom-extractor/tests/mapping/user_submission_activities/test_user_submission_activities.py:39]: ✔️

**Review by bradbanister — APPROVED** [review]
Looks good, thanks!

**Pull request merged**